tests/e2e/steps_consumer_misbehaviour.go

```diff
@@ -230,6 +230,10 @@ func stepsCauseConsumerMisbehaviour(consumerName string) []Step {
 						validatorID("alice"): 511,
 						validatorID("bob"):   20,
 					},
+					RepresentativePowers: &map[validatorID]uint{
```

Comment:
Using `RepresentativePowers` was the only way I found to verify that slashing indeed takes place.
Using `ValPowers` was not useful because after a validator gets jailed, the validator's voting power is `0`. I also could not `unjail` and then check `ValPowers` similarly to what is done in [steps_multi_consumer_downtime.go](https://github.com/cosmos/interchain-security/blob/67bb558fcb841b17a4f54fd2dbd191210ecd444f/tests/e2e/steps_multi_consumer_downtime.go#L338) because in contrast to a downtime test, for misbehaviour & equivocation we also we also tombstone the validator.

Reply:
Seems ok to me. Could rename it to something like StakedTokens instead of RepresentativePowers (looking at the code, I think that's what this actually is, right?)